source/index.txt

```diff
@@ -24,9 +26,9 @@ MongoDB Node Driver
 Introduction
 ------------
 
-Welcome to the documentation site for the official MongoDB Node.js 
-driver. You can add the driver to your application to work with MongoDB 
-in JavaScript. Download it using `npm <https://www.npmjs.com/>`__ 
+Welcome to the documentation site for the official MongoDB Node.js
+driver. You can add the driver to your application to work with MongoDB
```

Comment:
Suggestion:
Not related to the actual change, but seems like using the source constant here might help in the long run.

```suggestion
Welcome to the documentation site for the official {+driver-long+}. You can add the driver to your application to work with MongoDB
```